samples/repl-server/client.js

```diff
@@ -0,0 +1,19 @@
+const repl = require("repl");
+
+const myeval = async (cmd, ctx, fname, callback) => {
+  const res = await fetch('http://localhost:8080', {
```

Comment:
Might be more efficient to set this up as a single long-running fetch where each command is streamed in rather than a new fetch for every command. But for now, this works.

Reply:
Hmm, I wonder if that could allow us to stream stdout instead of having to batch it per-command. Will experiment with this.